packages/client/src/components/ProtectedPage.tsx

```diff
@@ -113,7 +91,15 @@ class ProtectedPageComponent extends React.Component<Props, IProtectPageState> {
     }
 
     newState.loading = false
```

Comment:
I benchmarked this and it shouldn't really take much longer than a couple of milliseconds to do the JSON.parse's of storage's getItems. Could only having the LoadingBar in "Page" suffice?